Source/SwiftLintFramework/Rules/ConditionalReturnsOnNewlineRule.swift

```diff
@@ -30,19 +30,21 @@ public struct ConditionalReturnsOnNewlineRule: ConfigurationProviderRule, Rule,
     )
 
     public func validate(file: File) -> [StyleViolation] {
-        let pattern = "(guard|if)[^\n]*return"
+        let pattern: String = configuration.ifOnly ? "(if)[^\n]*return" : "(guard|if)[^\n]*return"
+
         return file.rangesAndTokens(matching: pattern).filter { _, tokens in
             guard let firstToken = tokens.first, let lastToken = tokens.last,
                 SyntaxKind(rawValue: firstToken.type) == .keyword &&
                     SyntaxKind(rawValue: lastToken.type) == .keyword else {
                         return false
             }
 
-            return ["if", "guard"].contains(content(for: firstToken, file: file)) &&
+            let searchTokens: [String] = configuration.ifOnly ? ["if"] : ["if", "guard"]
```

Comment:
we usually don't add explicit types

Reply:
This was merely to speed up the compiler since it seems to be slow on type-checking ternary operators. But I can remove it, if you want.

Reply:
Removed.